plugins/Monolog/Test/Integration/LogTest.php

```diff
@@ -258,4 +248,27 @@ public static function getLogFileLocation()
     {
         return StaticContainer::get('path.tmp') . '/logs/piwik.test.log';
     }
+
+    public function provideContainerConfig()
+    {
+        return array(
+            'Psr\Log\LoggerInterface' => \DI\get('Monolog\Logger')
+        );
+    }
+
+    private function recreateLogSingleton($backend, $level = 'INFO')
+    {
+        $newEnv = new Environment('test', array(
+            'ini.log.log_writers' => array($backend),
+            'ini.log.log_level' => $level,
+            'ini.log.string_message_format' => self::STRING_MESSAGE_FORMAT,
+            'ini.log.logger_file_path' => self::getLogFileLocation(),
+            'Psr\Log\LoggerInterface' => \DI\get('Monolog\Logger')
```

Comment:
Could these 3 lines be moved to `provideContainerConfig()` or does the new environment completely ignores the current one? If that's the case, then why do we bind LoggerInterface in `provideContainerConfig()`? (isn't that useless?)